R/planning_helper.R

```diff
@@ -0,0 +1,56 @@
+# Setup for googledrive authentication. Set the appropriate env vars in
+# .Renviron and make sure you ran googledrive::drive_auth() interactively with
+# these settings for the first run (or to renew an expired Oauth token)
+if (Sys.getenv("GARGLE_OAUTH_EMAIL") != "") {
+  options(gargle_oauth_email = Sys.getenv("GARGLE_OAUTH_EMAIL"))
+}
+if (Sys.getenv("GARGLE_OAUTH_CACHE") != "") {
+  options(gargle_oauth_cache = Sys.getenv("GARGLE_OAUTH_CACHE"))
+}
+
```

Comment:
Note: I currently cannot test this, because my `gargle` is not configured. I will do so, and re-review.

Reply:
update: connection worked.